aspnetcore/fundamentals/minimal-apis/handle-errrors.md

```diff
@@ -8,9 +8,7 @@ ms.date: 5/2/2023
 uid: fundamentals/minimal-apis/handle-errors
 ---
 
-<!-- Can't add this until 8 is released and not-latest-version.md is updated to 8.0
 [!INCLUDE[](~/includes/not-latest-version.md)]
---> 
 
```

Comment:
This article now has 8.0 content, and while 6 and 7 versions are identical, if 6 or anything earlier is selected it automatically switches to 7, so you'll never get the "not latest" warning.